src/api/v3/single_page.py

```diff
@@ -0,0 +1,41 @@
+"""
+View to return a JSON representation of a single page. The page can
+be selected via the id or the permalink.
+"""
+from django.http import JsonResponse, HttpResponse
+from django.shortcuts import get_object_or_404
+
+from cms.models import Region
+from .pages import transform_page
+
+def single_page(request, region_slug, language_code):
+    """
+    View function returning the desired page as a JSON or a 404 if the
+    requested page does not exist.
+    Args:
+        request : The request that has been send to the Django.
+        region_slug : Slug defining the region
+        language_code : Code to identify the desired language.
+
+    Returns:
+        HttpResponse: Return an JSON with the requested page and a HTTP 200 or an 404.
```

Comment:
```suggestion

    :param request: The request that has been sent to the Django server
    :type request: django.http.HttpRequest

    :param region_slug: Slug defining the region
    :type region_slug: str

    :param language_code: Code to identify the desired language
    :type language_code: str

    :raises django.http.Http404: HTTP status 404 if the request is malformed or no page with the given id or url exists. 

    :return: Return a JSON with the requested page and a HTTP status 200.
    :rtype: django.http.JsonResponse
```
I'd suggest sticking to the official Sphinx docstring syntax as described in #333.